packages/apple-pay-integration/src/apple-pay-session-factory.ts

```diff
@@ -1,10 +1,4 @@
-interface ApplePayWindow extends Window {
-    ApplePaySession: ApplePaySession;
-}
-
-function isApplePayWindow(window: Window): window is ApplePayWindow {
-    return "ApplePaySession" in window;
-}
+import { isApplePayWindow, ApplePayWindow } from '@bigcommerce/checkout-sdk/payment-integration-api';
```

Comment:
All look good to be. If it won't be reused for payment methods other than ApplePay, maybe we can leave it in Apple Pay pkg?

Reply:
It's used here https://github.com/bigcommerce/checkout-sdk-js/blob/master/packages/core/src/payment/payment-method-action-creator.ts#L69